ml_metadata/tools/docker_server/Dockerfile.konflux

```diff
@@ -63,7 +63,7 @@ ENV GRPC_PORT "8080"
 ENV METADATA_STORE_SERVER_CONFIG_FILE ""
 
 # Introduces tzdata package here to avoid LoadTimeZone check failed error in the metadata store server.
-RUN microdnf install -y \
+RUN microdnf reinstall -y \
   tzdata
```

Comment:
this is a very strange change; does it help anything actually?

Reply:
@jiridanek found this article explaining why this change is needed https://access.redhat.com/solutions/5616681